aips/xchain_daa_authentication.md

```diff
@@ -0,0 +1,132 @@
+---
+aip: TBD
+title: x-chain DAA authentication using Sign-in-With-Solana
+author: igor-aptos, brian, 0xmaayan, lightmark
+Status: Draft
+type: Framework
+created: 04/08/2024
+---
+
+# AIP-X - (AIP title)
```

Comment:
This needs a proper title and number

Reply:
next one is currently aip-121 - https://github.com/aptos-foundation/AIPs/tree/main/aips

you need to rename the file as well, and create an issue and put a link to it in the template